tests/python/helpers.py

```diff
@@ -1,45 +1,197 @@
 import asyncio
+import logging
+import math
 from hashlib import sha256
+from typing import List
+
+import aiohttp
+
+from kin import Keypair, Environment
+from kin.blockchain.horizon import Horizon
 from kin.blockchain.builder import Builder
-from kin import Keypair
+from kin.config import SDK_USER_AGENT
 from kin_base import Keypair as BaseKeypair
 
-from concurrent.futures import ThreadPoolExecutor
+import kin
+import kin_base
+
+NETWORK_NAME = 'LOCAL'
+MIN_FEE = 100
+MAX_OPS = 100
+TX_SET_SIZE = 500
+
+
+def root_account_seed(passphrase: str) -> str:
+    """Return the root account seed based on the given network passphrase."""
+    network_hash = sha256(passphrase.encode()).digest()
+    return kin_base.Keypair.from_raw_seed(network_hash).seed().decode()
 
 
 def derive_root_account(passphrase):
-    """Return the keypair of the root account, based on the network passphrase"""
+    """Return the keypair of the root account, based on the network passphrase."""
     network_hash = sha256(passphrase.encode()).digest()
     seed = BaseKeypair.from_raw_seed(network_hash).seed().decode()
     return Keypair(seed)
 
 
-async def send_txs(txs):
-    with ThreadPoolExecutor(max_workers=len(txs)) as executor:
-        loop = asyncio.get_running_loop()
-        promises = [loop.run_in_executor(executor, tx.submit) for tx in txs]
+async def create_accounts(env: Environment, channels: List[Builder], root_account: BaseKeypair, starting_balance, horizon_addresses: List[str] = []):
+    """Asynchronously create accounts and return a transaction Builder instance for each account.
+
+    Each Builder instance uses a different Horizon endpoint if given a non-empty list.
+    """
+    accounts_to_create_num = len(channels) * MAX_OPS
+    logging.info('creating %d accounts', accounts_to_create_num)
+
+    accounts_to_create = [Keypair() for _ in range(accounts_to_create_num)]
+    xdrs = []
+    for i, channel in enumerate(channels):
+        for kp in accounts_to_create[i*MAX_OPS : i*(MAX_OPS) + MAX_OPS]:
+            # use the root account as the source of the operations
+            #
+            # NOTE channels need a starting balance since they need to pay fees because they are not prioritized
+            channel.append_create_account_op(source=root_account.address(),
```

Comment:
address() returns bytes, maybe you need to decode it ```root_account.address().decode()```

Reply:
it works like this, so i guess not